wiki/Tournaments/Official_support/en.md

```diff
@@ -233,12 +257,14 @@ Please do not submit a main menu banner with your original request — you will
 
 While support for advertising and sharing the outcome of your tournament is available via news posts, there are additional requirements that must be considered:
 
-- Only news posts advertising registrations and recapping of **final** results are presently permitted.
+- Only news posts advertising registrations and/or recapping of **final** results are presently permitted. Additional news posts are presently not permitted.
+  - Tournaments that are given permission to advertise registrations are also expected to provide a conclusionary news post recapping **final** results.
+  - If a news post advertising registrations is denied but all other [requirements](#requesting-in-game-banner-and-news-post-support) are met, a tournament may still be given permission for a conclusionary news post. Hosts and organisers interested in doing so should send in a new request after the tournament has concluded.
```

Comment:
if a tournament is to have a news post at all, it makes no sense to wait until it has concluded most of the time because coverage is time-sensitive (depending on the content)